src/main/java/hockeyapp/HockeyappRecorder.java

```diff
@@ -82,7 +78,6 @@
     @DataBoundConstructor
     public HockeyappRecorder(List<HockeyappApplication> applications, boolean debugMode,
                              BaseUrlHolder baseUrlHolder, boolean failGracefully) {
-        this.schemaVersion = SCHEMA_VERSION_NUMBER;
```

Comment:
From my memory, I think the schema version was serialised to the file but could not be read back in and this cause part of the issue